module/VuFind/tests/unit-tests/src/VuFindTest/AjaxHandler/CommentRecordTest.php

```diff
@@ -143,17 +171,34 @@ public function testEmptyQuery()
      */
     public function testSuccessfulTransaction()
     {
-        $user = $this->getMockUser();
+        $user = $this->getmockBuilder(User::class)
```

Comment:
Typo: getmockBuilder for getMockBuilder.

Reply:
Fixed.